### ISSUE TITLE
Issue in InnerClassList tooltip

### ISSUE THREAD

**Issue by andylizi** [Col-E/Recaf#87]
In [asm-tree/InnerClassNode.java#L39-52](https://gitlab.ow2.org/asm/asm/blob/19735cbf6d436d99fc15e3a2036a2746e3f1be80/asm-tree/src/main/java/org/objectweb/asm/tree/InnerClassNode.java#L39-52):
```java
/** The internal name of an inner class (see {@link org.objectweb.asm.Type#getInternalName()}). */
public String name;

/**
 * The internal name of the class to which the inner class belongs (see {@link
 * org.objectweb.asm.Type#getInternalName()}). May be {@literal null}.
 */
public String outerName;

/**
 * The (simple) name of the inner class inside its enclosing class. May be {@literal null} for
 * anonymous inner classes.
 */
public String innerName;
```
According to this, 
`outerName` is `the internal name of the class to which the inner class belongs` and 
`innerName` is `the simple name of the inner class inside its enclosing class`.

However, in [recaf/en.json#L217-L218](../blob/3ebc5c6401164097294a5c2165a021bdf9c0e09b/src/resources/lang/en.json#L217-L218):
```json
"ui.bean.class.innerclasses.inner.tooltip": "The internal name of the class to which the inner class belongs. May be null.",
"ui.bean.class.innerclasses.outer.tooltip": "The simple name of the inner class inside its enclosing class. May be null for anonymous inner classes",
```

**Comment by andylizi** [comment]
A simple fix:
```patch
From f6ee1dcf4d25f39d35a665b88b2479021ac490e2 Mon Sep 17 00:00:00 2001
From: andylizi <andylizi@qq.com>
Date: Fri, 28 Sep 2018 13:37:42 +0800
Subject: [PATCH] Fix a typo in translation. Closes #87

---
 src/resources/lang/en.json | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/resources/lang/en.json b/src/resources/lang/en.json
index 8a24cb9..0674fd9 100644
--- a/src/resources/lang/en.json
+++ b/src/resources/lang/en.json
@@ -214,8 +214,8 @@
 	"ui.bean.class.innerclasses.name": "Inner classes",
 	"ui.bean.class.innerclasses.desc": "List of inner classes.",
 	"ui.bean.class.innerclasses.name.tooltip": "The internal name of an inner class.",
-	"ui.bean.class.innerclasses.inner.tooltip": "The internal name of the class to which the inner class belongs. May be null.",
-	"ui.bean.class.innerclasses.outer.tooltip": "The simple name of the inner class inside its enclosing class. May be null for anonymous inner classes",
+	"ui.bean.class.innerclasses.outer.tooltip": "The internal name of the class to which the inner class belongs. May be null.",
+	"ui.bean.class.innerclasses.inner.tooltip": "The simple name of the inner class inside its enclosing class. May be null for anonymous inner classes",
 	"ui.bean.class.fields.name": "Fields",
 	"ui.bean.class.fields.desc": "List of fields.",
 	"ui.bean.class.methods.name": "Methods",
-- 
2.19.0.windows.1

```

**Comment by Col-E** [comment]
:+1: 
Nice catch
